src/store/mutations.ts

```diff
@@ -106,11 +108,13 @@ export const mutations: MutationTree<CalChartState> = {
     );
     state.selectedSS = state.show.stuntSheets.length - 1;
     state.beat = 0;
+    state.show.calculateWarnings();
   },
   [Mutations.DELETE_STUNT_SHEET](state): void {
     state.show.stuntSheets.splice(state.selectedSS, 1);
     state.selectedSS = Math.max(0, state.selectedSS - 1);
     state.beat = 0;
+    state.show.calculateWarnings();
```

Comment:
This file has some occurrences of `recurseWarnings`, and some of `calculateWarnings`. I see why you did this, but I found myself needing to do a double-take on some of the calls to `calculateWarnings` to remind myself that they are shallow calculations of warnings. Maybe something could be added to the name to emphasize the shallow nature, to differentiate it more obviously from `recurseWarnings`? Maybe something like `shallowCalculateWarnings` and `deepCalculateWarnings`?

Reply:
Thanks for the recommendation. shallow and deep more accurately describe what we're doing so that's what I'll go with!